src/seedu/addressbook/parser/generateHash.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.parser;
+
+import seedu.addressbook.commands.IncorrectCommand;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.lang.String;
+
+public class generateHash {
```

Comment:
Add comments explaining the hashing process. An important part of SE is to aid future developers easily understand your code.